base_python/python3.8/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8-slim as compile-image
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
```

Comment:
once we combine both commands I would also start with `set -euxo pipefail` to exit immediately when an error happens it's a best practice more info can be found here: https://bertvv.github.io/cheat-sheets/Bash.html